python/friesian/example/recall/search.py

```diff
@@ -114,7 +109,7 @@ def search(args):
     with StopWatch("do_search spark >>>>>>") as sw:
         df = spark.read.parquet(args.parquet_path)
         print('Total number of items: ', df.count())
-        rdd = df.rdd.repartition(num_executors)  # Each node runs one faiss task
+        rdd = df.rdd.repartition(args.num_repartition)  # Each node runs one faiss task
```

Comment:
num_partition should be the same as num_executors right?

Reply:
done